.github/workflows/cypress.yml

```diff
@@ -236,6 +236,11 @@ jobs:
         with:
           node-version: '14'
 
+      - name: Install Modules
+        uses: borales/actions-yarn@v2.3.0
+        with:
+          cmd: install 
```

Comment:
Could this just be a step that runs `yarn install`? Other jobs already do or will run it directly.

https://github.com/department-of-veterans-affairs/vets-website/pull/17786/files#diff-5da9a41590fb261de0e464c95862528100c42c77dffad53b7c6de228b6afd2bdL48-L51

Reply:
Good call. I updated that.